react/src/components/UserPrefModal.tsx

```diff
@@ -0,0 +1,300 @@
+import { useWebComponentInfo } from './DefaultProviders';
+import { passwordPattern } from './ResetPasswordRequired';
+import { useSuspendedBackendaiClient } from "../hooks";
+import { baiSignedRequestWithPromise } from '../helper';
+import { useTanMutation } from '../hooks/reactQueryAlias';
+
+const UserPrefModal : React.FC = () => {
+  const { t } = useTranslation();
+
+  const [form] = Form.useForm();
+
+  const [messageApi, contextHolder] = message.useMessage();
+  const _showSuccessMessage = (successMessage: string) => {
+    messageApi.open({
+      type: "success",
+      content: successMessage
+    });
+  };
+  const _showErrorMessage = (errorMessage: string) => {
+    messageApi.open({
+      type: "error",
+      content: errorMessage
+    })
+  }
+
+  const { value, dispatchEvent } = useWebComponentInfo();
+  let parsedValue: {
+    isOpen: boolean;
+    full_name: string;
+    userId: string;
+    totpSupported: boolean;
+    totpActivated: boolean;
+    loggedAccount: {
+      access_key: string;
+    };
+    keyPairInfo: {
+      keypairs: [{
+        access_key: string;
+        secret_key: string;
+      }];
+    }
+  };
+  try {
+    parsedValue = JSON.parse(value || "");
+  } catch (error) {
+    parsedValue = {
+      isOpen: false,
+      full_name: "",
+      userId: "",
+      totpSupported: false,
+      totpActivated: false,
+      loggedAccount: {access_key: ""},
+      keyPairInfo: {keypairs:[{access_key: "", secret_key: ""}]}
+    };
+  };
+  const { isOpen, full_name, userId, totpSupported, totpActivated, loggedAccount, keyPairInfo } = parsedValue;
+
+  let selectOptions: any[] = [];
+  if (keyPairInfo.keypairs) {
+    for (let i=0; i < keyPairInfo.keypairs.length; i++) {
+      selectOptions.push({value: keyPairInfo.keypairs[i].secret_key, label: keyPairInfo.keypairs[i].access_key})
+    }
+  };
+
+  const baiClient = useSuspendedBackendaiClient();
+  const mutationToUpdateUserFullName = useTanMutation({
+    mutationFn: (body: {
+      full_name: string;
+      email: string
+    }) => {
+      return baiSignedRequestWithPromise({
+        method: "POST",
+        url: `/auth/update-full-name`,
+        body,
+        client: baiClient,
+      });
+    }
+  });
+  const mutationToUpdateUserPassword = useTanMutation({
+    mutationFn: (body: {
+      old_password: string;
+      new_password: string;
+      new_password2: string;
+    }) => {
+      return baiSignedRequestWithPromise({
+        method: "POST",
+        url: `/auth/update-password`,
+        body,
+        client: baiClient,
+      });
+    }
+  });
+  
+  useEffect(()=> {
+    for(let i=0; i<selectOptions.length; i++) {
+      if (loggedAccount.access_key === selectOptions[i].label) {
+        form.setFieldsValue({
+          access_key: selectOptions[i].label,
+          secret_key: selectOptions[i].value
+        })
+        break;
+      }
+    }
+    form.setFieldsValue({
+      full_name: full_name
+    });
+  });
```

Comment:
How about using `initialValues`? ([ref](https://ant.design/components/form))

Reply:
@Sujin-Kim1
Thanks for the review!
I have question this review.
I think input of full_name is correct to use initialValue according to your review.
But access_key and secret_key is correct to use setFieldsValue. Because access_key need to check if it same loggedAccount. And secret_key is pair of access_key.
is it correct to use setFieldsValue??🧐

Reply:
Ah, I didn't consider this uses a select component. Then, with reference this code, set `secret_key` automatically.   https://github.com/lablup/backend.ai-webui/blob/51852efbb696c20e94a1055fe6db85e234e85eaf/react/src/components/VFolderSelect.tsx#L86-L97

Reply:
@Sujin-Kim1
Thank you😍